csrankings.csv

```diff
@@ -16503,7 +16504,7 @@ Ryan Farrell,Brigham Young University,https://faculty.cs.byu.edu/~farrell,56EZh6
 Ryan Hayward,University of Alberta,https://webdocs.cs.ualberta.ca/~hayward,h-VFYuUAAAAJ
 Ryan Henry,University of Calgary,http://homes.soic.indiana.edu/henry,kj6ctNwAAAAJ
 Ryan J. Leduc,McMaster University,http://www.cas.mcmaster.ca/~leduc,YFaGsQYAAAAJ
-Ryan K. L. Ko,University of Queensland,https://ryanko.org/bio,doBtsCkAAAAJ
+Ryan K. L. Ko,University of Queensland,https://researchers.uq.edu.au/researcher/23784,doBtsCkAAAAJ
 Ryan Kastner,University of California - San Diego,http://kastner.ucsd.edu,lfLiQmcAAAAJ
 Ryan Kok Leong Ko,University of Queensland,https://ryanko.org/bio,doBtsCkAAAAJ

```

Comment:
This home page also needs to be updated (don't delete this entry - both are needed).